test/integration/migrate/index.js

```diff
@@ -72,6 +72,22 @@ module.exports = function(knex) {
       });
     }
 
+    if (knex.client.driverName === 'sqlite3') {
+      it('should not fail rename-and-drop-column with multiline sql from legacy db', () => {
+        return knex.migrate
```

Comment:
Could you please use async/await instead?

Reply:
Sure.